src/doc/unstable-book/src/language-features/asm-experimental-arch.md

```diff
@@ -77,7 +77,8 @@ This feature tracks `asm!` and `global_asm!` support for the following architect
 | PowerPC      | `reg`                           | None           | `i8`, `i16`, `i32`, `i64` (powerpc64 only) |
 | PowerPC      | `reg_nonzero`                   | None           | `i8`, `i16`, `i32`, `i64` (powerpc64 only) |
 | PowerPC      | `freg`                          | None           | `f32`, `f64`                            |
-| PowerPC      | `vreg`                          | N/A            | Only clobbers                           |
+| PowerPC      | `vreg`                          | `altivec`      | `vector_bool_char`, `vector_bool_short`, `vector_bool_int`, `vector_signed_char`, `vector_signed_short`, `vector_signed_int`, `vector_unsigned_char`, `vector_unsigned_short`, `vector_unsigned_int`, `vector_float` |
```

Comment:
Can you use the same `iNxM` type names that we use for other platforms for consistency? Alternatively we could change the documentation in the reference to explicitly say which types in stdarch are supported, but that's not exactly true since we currently also accept types from std::simd.

Reply:
Updated to use `iNxM`/`fNxM`.